frontend/src/data/config/MetadataMap.ts

```diff
@@ -536,6 +537,27 @@ export const dataSourceMetadataMap: Record<DataSourceId, DataSourceMetadata> = {
     downloadable: true,
     time_period_range: '2001 - current',
   },
+  maternal_health: {
+
+    id: "maternal_health",
+    data_source_name: `JAMA`,
+    data_source_acronym: 'JAMA',
+    data_source_pretty_site_name: '',
+    data_source_link: '',
```

Comment:
let's use:
data_source_name: 'Trends in State-Level Maternal Mortality by Racial and Ethnic Group in the United States'
data_source_acronym: 'JAMA'
data_source_pretty_: 'JAMA Network'
data_source_link: 'https://jamanetwork.com/journals/jama/fullarticle/2806661'